datastream-server-restli/src/main/java/com/linkedin/datastream/server/dms/DatastreamStore.java

```diff
@@ -21,6 +21,16 @@
    */
   Datastream getDatastream(String key);
 
+  /**
+   * Retrieves raw data in json from the given path.
+   */
+  String getRawData(String path);
+
+  /**
+   * Retrieves connector task path specific to the store implementation.
+   */
+  String getConnectorTaskPath(String connector, String task);
```

Comment:
Why do you need this? Can you not use KeyBuilders directly?

Reply:
Umm no, since DatastreamResources should only depend upon the DatastreamStore Iface and not on something which implements it, like zk. that's why I made helper functions in the store Iface which helps retrieve zk- related data.